### PR TITLE
Fixed RCM for 3.7

### PR DESCRIPTION
**Pull request by cfchase**
Updated `./reset_environment.sh  --rcm` to use 3.7 and dev defaults for now.

### CODE REVIEW

**Comment by cfchase** [comment]
[(3) catasb --rcm](https://trello.com/c/p7sFd36L/536-3-catasb-rcm)

File: config/my_vars.yml.example [hunk 1]
```diff
@@ -39,7 +42,23 @@ dockerhub_org: ansibleplaybookbundle
...
+## Override and use latest images pushed to developer servers to mimic stage
+#
+# broker_registry_url: http://registry.access.stage.redhat.com
+# origin_image_name: registry.ops.openshift.com/openshift3/ose
+#
+# - Dev Staging
+# broker_registry_url: http://asb-registry.usersys.redhat.com:5000
+# origin_image_name: asb-registry.usersys.redhat.com:5000/openshift3/ose
+
+
```
Comment (jwmatthews) [inline config/my_vars.yml.example]: What do you think about leaving the rcm specifics in rcm.yml?

so removing lines ~45-58?

**Comment by jwmatthews** [comment]
From what I can test this is working, as in the correct images were pulled down.  I can't do a full verification as we have a few issues to address related to recent service-catalog changes.

Ran on linux and saw cluster come up and expected images are pulled:
$ docker images
REPOSITORY                                                                   TAG                 IMAGE ID            CREATED             SIZE
docker.io/projecthelios/aws-service-broker                                   latest              45a5ddc607c3        5 hours ago         610.7 MB
asb-registry.usersys.redhat.com:5000/openshift3/ose-service-catalog          latest              fed05187de44        14 hours ago        339.2 MB
asb-registry.usersys.redhat.com:5000/openshift3/ose-ansible-service-broker   latest              5f6670f7164a        15 hours ago        622.2 MB
asb-registry.usersys.redhat.com:5000/openshift3/ose-haproxy-router           latest              afd0b19d3757        15 hours ago        1.072 GB
asb-registry.usersys.redhat.com:5000/openshift3/ose-sti-builder              latest              e3cb3c85d3e4        15 hours ago        1.053 GB
asb-registry.usersys.redhat.com:5000/openshift3/ose                          latest              cc43773ba9b2        16 hours ago        1.053 GB
asb-registry.usersys.redhat.com:5000/openshift3/ose-docker-registry          latest              d68e3512dc9e        16 hours ago        467.2 MB
asb-registry.usersys.redhat.com:5000/openshift3/ose-pod                      latest              7377140eb543        16 hours ago        208.7 MB
asb-registry.usersys.redhat.com:5000/openshift3/ose-deployer                 latest              a9f37ebc8c1e        26 hours ago        1.051 GB
quay.io/coreos/etcd                                                          latest              e8325015c810        5 days ago          35.73 MB
registry.access.redhat.com/rhel7/etcd                                        latest              93e8e4932fa2        6 weeks ago         238.6 MB


I don't see APBs present in Service Catalog, but think it's related to the broker I ran not including this fix:
 https://github.com/openshift/ansible-service-broker/pull/480


Below is the error I saw from service catalog

E1011 22:07:47.354578       1 controller_broker.go:643] Error updating ready condition for ServiceBroker "ansible-service-broker": Operation cannot be fulfilled on servicebrokers "ansible-service-broker": the object has been modified; please apply your changes to the latest version and try again
I1011 22:07:47.354606       1 controller.go:232] Error syncing ServiceBroker ansible-service-broker: Operation cannot be fulfilled on servicebrokers "ansible-service-broker": the object has been modified; please apply your changes to the latest version and try again

File: config/my_vars.yml.example (hunk 1 above)
Comment (cfchase) [reply config/my_vars.yml.example]: sure

**Comment by jwmatthews** [comment]
Re-tested and see Broker is up and using correct images.

Still don't see APBs from Service Catalog, but I think it's related to the downstream build not having the fix for https://bugzilla.redhat.com/show_bug.cgi?id=1500048

This PR is working as intended, will merge so we can continue testing.  Issues I'm encountering I believe are all related to downstream builds missing fixes we need.

**Pull request merged**